docs/sw.js

```diff
@@ -0,0 +1,25 @@
+var cacheName = 'egui-template-pwa';
+var filesToCache = [
+  './',
+  './index.html',
+  './egui_template_bg.wasm'
```

Comment:
Shouldn't `egui_template.js` be in here too?

Reply:
Makes sense, good call.